src/components/Book.vue

```diff
@@ -0,0 +1,43 @@
+<template lang="pug">
+  div.book-container
+    img.book-image(:src="book.image_url")
+    h3.book-title {{ book.title }}
+    span.book-author {{ book.author }}
+
+  </template>
+
+<script>
+export default {
+  name: 'Book',
+  props: {
+    book: Object
```

Comment:
define a default value for this

Reply:
But this prop is required, I don't think it should have a default value. I forgot to set it as required